utils/rpms/daos.spec

```diff
@@ -180,6 +181,7 @@ Requires: fuse3 >= 3
 Requires: fuse < 3, fuse3-libs >= 3.4.2
 %endif
 %endif
+Requires: numactl
```

Comment:
Looks like the ticket was updated to just prescribe this for `daos_admin`.

As an aside, if this really were needed for both client and server it would be better added as a requirement to the main `daos` package.

Reply:
removed entry in client package section